Makefile

```diff
@@ -22,7 +23,7 @@ install-build-deps: ## Install dependencies (packages and tools)
 
 .PHONY: build
 build: generate-bindata-assets generate-kubernetes-types  ## Build eksctl
-	CGO_ENABLED=0 go build -tags "$(GO_BUILD_TAGS)" -ldflags "-X $(version_pkg).gitCommit=$(git_commit) -X $(version_pkg).builtAt=$(built_at)" ./cmd/eksctl
+	CGO_ENABLED=0 time go build -v -ldflags "-X $(version_pkg).gitCommit=$(git_commit) -X $(version_pkg).builtAt=$(built_at)" ./cmd/eksctl
```

Comment:
should we remove the `time` before merging?

Reply:
Maybe, I would like to leave them there until I find the culprit though

Reply:
Modern build system measure time for you... Just saying ;)

I'm happy with make for this project, mostly because it's what everyone is familiar with.